docs/getting-started/faq.rst

```diff
@@ -53,33 +53,27 @@ for more information.
 Can I develop and sell closed source proprietary code that is a derivative of PyMedPhys?
 ----------------------------------------------------------------------------------------
 
-Not under PyMedPhys' current license you can't however the
-contributors of PyMedPhys may be open to relicensing on a case-by-case basis.
-This however may come at a fee, and if it does come at a fee there would need to be
-a structure created that enables all contributors and stake-holders of the
+No, the current PyMedPhys' license prohibits this. **However**, the
+PyMedPhys contributors may be open to re-licensing on a case-by-case basis.
+This would require negotiation and, likely, a suitable financial arrangement. Such an 
+arrangement would need to be structured to enable all contributors and stake-holders of the
 code in question to be fairly and proportionally represented.
 
-In reality relicensing PyMedPhys goes against its aim to create an open
-toolbox that we can all have access to and build upon, sharing the improvements
-with each other. But please don't let the fear of non-commercial viability stop you
-using the code, instead just contact us at (developers@pymedphys.com) and we may
-be able to come to an agreement.
```

Comment:
I'd prefer to omit the "goes against the aim to create an open toolbox". For example, depending on the arrangement, a closed source derivative might actually support/enhance the open toolbox. Moreover, I'd prefer not to dis-incentivise approaches in any way and simply consider them on a case-by-case basis.

Reply:
Yup that's fair. Can you have a look at the most recent version of that section. I did a rework the other day of that section that I think makes this part a little neater. Might be worth doing a pull and conflict resolution...